jax/experimental/jax2tf/jax2tf.py

```diff
@@ -133,29 +161,36 @@ def doit():
 
 
 def _interpret_fun(fun: lu.WrappedFun,
-                   in_vals: Sequence[TfVal]) -> Sequence[TfVal]:
+                   in_vals: Sequence[TfValOrUnit]) -> Sequence[TfValOrUnit]:
   with core.new_master(TensorFlowTrace) as master:
     fun = _interpret_subtrace(fun, master)
-    out_vals = fun.call_wrapped(*in_vals)
+    out_vals: Sequence[TfValOrUnit] = fun.call_wrapped(*in_vals)
     del master
   return out_vals
 
 
 @lu.transformation
-def _interpret_subtrace(master: core.MasterTrace, *in_vals: TfVal):
+def _interpret_subtrace(master: core.MasterTrace, *in_vals: TfValOrUnit):
   trace = TensorFlowTrace(master, core.cur_sublevel())
-  in_tracers = [TensorFlowTracer(trace, val) for val in in_vals]
-  outs = yield in_tracers, {}
-  out_tracers = map(trace.full_raise, outs)
-  out_vals = [t.val for t in out_tracers]
+  in_tracers = tuple(TensorFlowTracer(trace, val) for val in in_vals)
+  outs = yield in_tracers, {}  # type : Sequence[TfValOrUnit]
+  out_tracers: Sequence[TensorFlowTracer] = map(trace.full_raise, outs)  # type: ignore[assignment]
```

Comment:
The type here is a lie since `map` returns a generator? Would be easier to type that instead (or not type).

Reply:
I applied `util.safe_map`, which returns a sequence. I found the types helpful even for myself to keep track of what is what, I think that they are useful documentation, so I'd rather keep them